x-pack/legacy/plugins/uptime/public/state/effects/index.ts

```diff
@@ -6,7 +6,9 @@
 
 import { fork } from 'redux-saga/effects';
 import { fetchMonitorDetailsEffect } from './monitor';
+import { fetchSnapshotCountSaga } from './snapshot';
 
-export function* rootEffect() {
+export function* rootSaga() {
```

Comment:
For me i wanted to rename it  to effect instead of saga, i feel effect is more common term to describe this kind of behavior.

Reply:
👍 this was more merge confusion

Reply:
Resolved in https://github.com/elastic/kibana/pull/48035/commits/0f5ba99f5ee01cd908a9202b9b7b6a5d2cc8c84f.